diff --git a/dist/node/chunks/dep-D4NMHUTW.js b/dist/node/chunks/dep-D4NMHUTW.js
index 5b079dae3f59da47989295ba58657c171eab0d27..fad925995badda420b4f5aae893921154e02cb92 100644
--- a/dist/node/chunks/dep-D4NMHUTW.js
+++ b/dist/node/chunks/dep-D4NMHUTW.js
@@ -6935,19 +6935,23 @@ function optimizeSafeRealPathSync() {
       return;
     }
   }
-  exec("net use", (error, stdout) => {
-    if (error) return;
-    const lines = stdout.split("\n");
-    for (const line of lines) {
-      const m = parseNetUseRE.exec(line);
-      if (m) windowsNetworkMap.set(m[2], m[1]);
-    }
-    if (windowsNetworkMap.size === 0) {
-      safeRealpathSync = fs__default.realpathSync.native;
-    } else {
-      safeRealpathSync = windowsMappedRealpathSync;
-    }
-  });
+  try {
+    exec("net use", (error, stdout) => {
+      if (error) return;
+      const lines = stdout.split("\n");
+      for (const line of lines) {
+        const m = parseNetUseRE.exec(line);
+        if (m) windowsNetworkMap.set(m[2], m[1]);
+      }
+      if (windowsNetworkMap.size === 0) {
+        safeRealpathSync = fs__default.realpathSync.native;
+      } else {
+        safeRealpathSync = windowsMappedRealpathSync;
+      }
+    });
+  } catch (error) {
+    // Ignore restricted environment errors.
+  }
 }
 function ensureWatchedFile(watcher, file, root) {
   if (file && // only need to watch if out of root
diff --git a/dist/node-cjs/publicUtils.cjs b/dist/node-cjs/publicUtils.cjs
index e3e58c10082d74bf845a2c97cea12dbca740bfb9..abeaa60ab3638b5d659e4641d593e403d13a8e29 100644
--- a/dist/node-cjs/publicUtils.cjs
+++ b/dist/node-cjs/publicUtils.cjs
@@ -1271,19 +1271,23 @@ function optimizeSafeRealPathSync() {
       return;
     }
   }
-  node_child_process.exec("net use", (error, stdout) => {
-    if (error) return;
-    const lines = stdout.split("\n");
-    for (const line of lines) {
-      const m = parseNetUseRE.exec(line);
-      if (m) windowsNetworkMap.set(m[2], m[1]);
-    }
-    if (windowsNetworkMap.size === 0) {
-      safeRealpathSync = fs$1.realpathSync.native;
-    } else {
-      safeRealpathSync = windowsMappedRealpathSync;
-    }
-  });
+  try {
+    node_child_process.exec("net use", (error, stdout) => {
+      if (error) return;
+      const lines = stdout.split("\n");
+      for (const line of lines) {
+        const m = parseNetUseRE.exec(line);
+        if (m) windowsNetworkMap.set(m[2], m[1]);
+      }
+      if (windowsNetworkMap.size === 0) {
+        safeRealpathSync = fs$1.realpathSync.native;
+      } else {
+        safeRealpathSync = windowsMappedRealpathSync;
+      }
+    });
+  } catch (error) {
+    // Ignore restricted environment errors.
+  }
 }
 function arraify(target) {
   return Array.isArray(target) ? target : [target];
